migrations/0027_change_query_schedule_to_text.py

```diff
@@ -0,0 +1,9 @@
+from redash.models import db
+import peewee
+from playhouse.migrate import PostgresqlMigrator, migrate
+
+if __name__ == '__main__':
+    migrator = PostgresqlMigrator(db.database)
+    with db.database.transaction():
+        db.database.execute_sql('ALTER TABLE queries ALTER COLUMN schedule TYPE text;')
```

Comment:
1) because the crontab string could grow very long (and I don't exactly know how long)
2) because text and char have the same performances (read the tip from https://www.postgresql.org/docs/9.1/static/datatype-character.html)